src/test/java/com/google/plantasy/TimingPropertiesTest.java

```diff
@@ -225,38 +219,38 @@ public void giveUserQuizTakenPoints_falseUserQuizStatus(){
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         Entity user = new Entity("user");
- 
+
         boolean actual = timing_properties_test.giveUserQuizTakenPoints(false, user, datastore);
         Assert.assertEquals(false, actual);
     }
 
-    @Test
+    @Test(expected = NullPointerException.class)
     //Tests if user does not have a score property 
     public void giveUserQuizTakenPoints_notScoreProperty() {
         QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        
+
         Entity user = new Entity("user");
         user.setProperty("quiz_timestamp", 159430944365L);
         datastore.put(user);
- 
+
         boolean actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
-        Assert.assertEquals(user.getProperty("score"), 20);
+        user.setProperty("score", 20);
```

Comment:
I'm a bit confused by this test case. We're asserting that there will be a null pointer exception, right? Will that most likely happen on line 237? If so, do we need line 238?

Reply:
Oliver mentioned this too, I will get rid of line 238. I added it because this test means the user does not have a score entity so I was going to add it here